tests/conftest.py

```diff
@@ -25,27 +25,120 @@ def fake_get_header(self, request):
 ForcedAuthentication.get_header = fake_get_header
 
 USER_ID = random_id()
+USER_2_ID = random_id()
+SHIPPER_ID = random_id()
 ORGANIZATION_ID = random_id()
+ORGANIZATION_2_ID = random_id()
 VAULT_ID = random_id()
 TRANSACTION_HASH = 'txHash'
 DEVICE_ID = random_id()
 
 
-@pytest.fixture(scope='session')
-def token():
-    return get_jwt(username='user1@shipchain.io', sub=USER_ID, organization_id=ORGANIZATION_ID)
+if test_settings.PROFILES_ENABLED:
```

Comment:
I think all of these conftest definitions should exist in the profiles_enabled module instead of having a conditional here